packages/components/psammead-media-player/src/Amp/index.jsx

```diff
@@ -13,7 +13,15 @@ const AmpHead = () => (
   </Helmet>
 );
 
-const AmpMediaPlayer = ({ src, placeholderSrc, placeholderSrcset, title }) => {
+const AmpMediaPlayer = ({
+  src,
+  placeholderSrc,
+  placeholderSrcset,
+  title,
+  alt,
```

Comment:
I don't think we need a new `alt` prop, it can be hardcoded to `""` on L41 for consistency with https://github.com/bbc/psammead/pull/2476